ci_framework/plugins/action/ci_kustomize.py

```diff
@@ -0,0 +1,755 @@
+#!/usr/bin/env python3
+
+# Copyright Red Hat, Inc.
+# Apache License Version 2.0 (see LICENSE)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+action: ci_kustomize
+
+short_description: Applies a set of k8s Kustomizations to a set of manifests
+
+description:
+- Allows applying a set of Kustomizations to a given set of manifests using
+  the kustomize or oc tools.
+- kustomize or oc should be discoverable by in the PATH.
+- This modules takes a set of manifest files pointed by I(target_path) and
```

Comment:
this "I(" must be part of module documentation generation? I see it in a few places here

Reply:
`I`is supposed to reference a input arg, that's what docs say, but I haven't checked the generated doc.

Reply:
(not sure we generate any doc out of those kind of content, to be fair)

Reply:
thank you